qa/L0_backend_python/examples/test.sh

```diff
@@ -64,7 +67,7 @@ fi
 grep "PASS" add_sub_client.log
 if [ $? -ne 0 ]; then
     echo -e "\n***\n*** Failed to verify pytorch example. \n***"
-    cat pytorch_client.log
+    cat add_sub_client.log
```

Comment:
Define a variable CLIENT_LOG before running each example and then use that in those example section.